api/app/src/providers/withings.ts

```diff
@@ -303,6 +303,10 @@ export class Withings extends Provider implements OAuth2 {
       params,
     });
 
+    if (response.data.error) {
```

Comment:
`response.data` is of type `any`, meaning it could be null/undefined/etc

As such, `.error` could throw an error.

Using `if (response.data?.error)` will take care of this scenario